script_vts_1.py

```diff
@@ -0,0 +1,62 @@
+"""import of modules"""
+
+import argparse
+import os
+import sys
+from pathlib import Path
+
+if __name__ == '__main__':
+
+    # Create the parser to have arguments in the script
+    parser = argparse.ArgumentParser()
+
+    # Adding SHA argument (let required True for now?)
+    parser.add_argument('--SHA', type=str, required=True)
```

Comment:
👍 I think having it required is a good idea for now.
Let's look at adding a second flag for passing (multiple) SHAs in from a file later on.

Reply:
adding it right now to the comment and working on it ASAP!